sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -58,6 +58,20 @@ def __init__(self, endpoint, credential, **kwargs):
             **kwargs
         )
 
+    def __enter__(self):
```

Comment:
changos... I actually don't know what this it :( will prefer someone from Python team to look at review